core/tests/protractor_utils/general.js

```diff
@@ -24,6 +24,12 @@ var dragAndDropScript = require('html-dnd').code;
 var action = require('../protractor_utils/action.js');
 
 var dragAndDrop = async function(fromElement, toElement) {
+  waitFor.visibilityOf(
+    fromElement,
+    'fromElement taking too long to load');
+  waitFor.visibilityOf(
+    toElement,
+    'toElement taking too long to load');
```

Comment:
Do we need `await` in these statements?

Reply:
I don't think we need an await here. However, @U8NWXD had recommended these changes. Can you please tell why?

Reply:
`waitFor.visibilityOf` is an async method and therefore needs to be awaited (see other usages in the codebase).

Reply:
Thanks! I got it. We will need the await statement.